k8s-info-cron-service/templates/cronjob.yaml

```
@@ -1,4 +1,4 @@
-{{- if .Values.enabled }}
+{{- if (and .Values.enabled (eq (include "k8s-info-cron-service.k8sVersionCheck" .) "true")) }}
```

This will essentially "fail silently" if kube version is not supported. Can you add a condition to do something like: 

~~~
{{- if eq (include "k8s-info-cron-service.k8sVersionCheck" .) "false")
  {{- fail "Unsupported kubernetes version" }}
{{- else  if .Values.enabled }}
...
~~~

---

Done